packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -51,13 +52,39 @@ SelfUtils.getSelves = (oldSelf, newSelf, deviceId) => {
   updates.moderatorChanged = SelfUtils.moderatorChanged(previous, current);
   updates.isMediaInactiveOrReleased = SelfUtils.wasMediaInactiveOrReleased(previous, current);
 
+  updates.isMediaInactive = SelfUtils.isMediaInactive(previous, current);
+
   return {
     previous,
     current,
     updates
   };
 };
 
+SelfUtils.isMediaInactive = (previous, current) => {
+  if (previous && previous.joinedWith && previous.joinedWith.mediaSessions &&
+     current && current.joinedWith && current.joinedWith.mediaSessions) {
+    const previousMediaStatus = SelfUtils.getMediaStatus(previous.joinedWith.mediaSessions);
+    const currentMediaStatus = SelfUtils.getMediaStatus(current.joinedWith.mediaSessions);
+
+    if (previousMediaStatus.audio.state !== MEDIA_STATE.inactive && currentMediaStatus.audio.state === MEDIA_STATE.inactive && currentMediaStatus.audio.direction !== MEDIA_STATE.inactive) {
+      return true;
+    }
+
+    if (previousMediaStatus.audio.state !== MEDIA_STATE.inactive && currentMediaStatus.audio.state === MEDIA_STATE.inactive && currentMediaStatus.audio.direction !== MEDIA_STATE.inactive) {
```

Comment:
Why is this line repeated three times?

Reply:
my bad it a copy past mistake . It should be audio video and share